### PR TITLE
RF R1 8-9Xu PCB

### PR DESCRIPTION
**Pull request by Cipulot**
Addition of the RF R1 8-9Xu PCB firmware. Open source universal PCB for the Realforce R1 family of keyboards.
## Description

The RF R1 8-9Xu PCB is a replacement PCB for the Realforce R1 family of keyboards. All the layouts from the R1 series are supported in a single PCB.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cipulot/rf_r1_8_9xu/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# RF R1 8-9Xu
+
+![RF_R1_8-9Xu PCB](https://i.imgur.com/lyy5OPr.png)
```
Comment (keyboard-magpie) [inline keyboards/cipulot/rf_r1_8_9xu/readme.md:3]:
```suggestion
![RF_R1_8-9Xu PCB](https://i.imgur.com/lyy5OPrh.png)
```
Same deal.

File: keyboards/cipulot/rf_r1_8_9xu/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Open source universal PCB for the Realforce R1 family of keyboards.
+
+* Keyboard Maintainer: [cipulot](https://github.com/cipulot)
+* Hardware Supported: RF_R1_8-9Xu
+* Hardware Availability: [Github](https://github.com/Cipulot/RF_R1_8-9Xu)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make rf_r1_8_9xu:default
```
Comment (keyboard-magpie) [inline keyboards/cipulot/rf_r1_8_9xu/readme.md:13]:
```suggestion
    make cipulot/rf_r1_8_9xu:default
```
Presumably?

File: keyboards/cipulot/rf_r1_8_9xu/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Flashing example for this keyboard:
+
+    make rf_r1_8_9xu:default:flash
```
Comment (keyboard-magpie) [inline keyboards/cipulot/rf_r1_8_9xu/readme.md:17]:
```suggestion
    make cipulot/rf_r1_8_9xu:default:flash
```
Yep.

**Review by keyboard-magpie — CHANGES_REQUESTED** [review]

File: keyboards/cipulot/rf_r1_8_9xu/readme.md (hunk 3 above)
Comment (Cipulot) [reply keyboards/cipulot/rf_r1_8_9xu/readme.md]: Overlook on my side. Changes applied 👍

File: keyboards/cipulot/rf_r1_8_9xu/readme.md (hunk 1 above)
Comment (Cipulot) [reply keyboards/cipulot/rf_r1_8_9xu/readme.md]: Done 👍

**Review by keyboard-magpie — APPROVED** [review]

File: keyboards/cipulot/rf_r1_8_9xu/rules.mk [hunk 4]
```diff
@@ -0,0 +1,2 @@
+CUSTOM_MATRIX = lite
+SRC += analog.c matrix.c ec_switch_matrix.c
```
Comment (drashna) [inline keyboards/cipulot/rf_r1_8_9xu/rules.mk:2]:
```suggestion
QUANTUM_LIB_SRC += analog.c 
SRC += matrix.c ec_switch_matrix.c
```

Comment (Cipulot) [reply keyboards/cipulot/rf_r1_8_9xu/rules.mk]: Done 👍

**Review by drashna — APPROVED** [review]

**Pull request merged**